canvas_course_admin_tools/settings/local.py

```diff
@@ -26,6 +26,15 @@
     'https://icommons-tools.dev.tlt.harvard.edu/course_conclusion'
 )
 
+CLASS_ROSTER = {
+    'sis_roster': {
+        'base_path': SECURE_SETTINGS.get('sis_roster_base_path', 'psp/hrvcsint'),
+        'static_path': '/EMPLOYEE/HRMS/c/SA_LEARNING_MANAGEMENT.SS_CLASS_ROSTER.GBL',
+        'base_url': SECURE_SETTINGS.get('sis_roster_base_url', 'https://csint.my.harvard.edu/'),
```

Comment:
This and the `base_path` settings are fine if you want the ability to go back-and-forth between hitting the production and staging/test endpoints of the tool, but if you are only ever going to go against the stage endpoint then you could alternatively hardcode those settings.